include/meson.build

```diff
@@ -47,6 +46,10 @@ if get_option('video')
   headers += files('exiv2/asfvideo.hpp', 'exiv2/matroskavideo.hpp', 'exiv2/quicktimevideo.hpp', 'exiv2/riffvideo.hpp')
 endif
 
+if get_option('webready')
```

Comment:
should be curl_dep.found()

Reply:
Hum, may I ask why? It seems the http files have nothing to do with curl